cmd/search/main.go

```diff
@@ -135,31 +143,109 @@ func (o *options) Run() error {
 			}
 		}
 
+		var deckURL *url.URL
+		client := &http.Client{
+			Timeout: 30 * time.Second,
+			Transport: &http.Transport{
+				TLSHandshakeTimeout: 10 * time.Second,
+				MaxConnsPerHost:     10,
```

Comment:
This seems like the only divergence from [`DefaultTransport`][1].  Do we actually care, or can we leave `Transport` unset with:

```go
client := &http.Client{Timeout: 30 * time.Second}
```

[1]: https://golang.org/pkg/net/http/#RoundTripper

Reply:
We run 5 in parallel and so we need to tolerate enough connections if the server or proxy in between us is limited so that we can stay parallel.  I.e. we want at least ~2x max parallelism